server/src/main/resources/application.yml

```diff
@@ -14,7 +14,7 @@ logging:
 
 tankobon:
   database:
-    file: \${tankobon.config-dir}/app_database.sqlite
+    file: \${tankobon.config-dir}/database.sqlite
```

Comment:
Can you undo the change in the `application-localdb.yml` as well, please? It's the only thing missing.